amundsen_application/static/js/components/Footer/tests/index.spec.tsx

```diff
@@ -0,0 +1,71 @@
+import * as React from 'react';
+import moment from 'moment-timezone';
+import { mount } from 'enzyme';
+
+import { Footer, FooterProps, mapDispatchToProps, mapStateToProps } from '../';
+
+import globalState from 'fixtures/globalState';
+
+/* TODO: Issues with mocking moment-timezone, all tests are failing
```

Comment:
I believe `Jest` will default to using UTC while the tests cases assumes that `moment.local` will use "PT".

You could add a timezone to `config.ts` and then use it for both the component and the tests. As a side effect, users will see a single timezone for a given deployment, but I think that is fine to say data was indexed relative to server time.

Reply:
The problem I had was beyond mocking the timezone itself, I was ideally trying to mock the object and methods we use because we don't actually need to execute moment's code. As discussed offline I'll revisit this as a separate task given that it won't affect our code coverage goal until we attempt to hit perhaps 90%+